Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM golang:1.14.4-buster
+# Note: when updating the go minor version here, also update the go-channel in snap/snapcraft.yml
+FROM golang:1.14.4-buster 
```

Comment:
`1.14.5` is a thing if it matters

Reply:
Good spot, but my change here is just a note to keep the minor version here in sync with the new snapcraft.yml, so is an unrelated change for this PR.